portfolio/src/main/java/com/google/sps/servlets/TranslateServlet.java

```diff
@@ -0,0 +1,32 @@
+package main.java.com.google.sps.servlets;
```

Comment:
It looks like this file and the SpanishServlet.java file are duplicates of each other. Let's keep one and delete the other.

Reply:
Ya it was an accident and I did not mean to push it to the github.